### PR TITLE
Update examples to remove configuration

### PR DESCRIPTION
**Pull request by msweeneydev**
This PR updates examples to remove configuration.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,14 +0,0 @@
-# Learn how to add code owners here:
-# https://help.github.com/en/articles/about-code-owners
-
-*                         @timothyis
```
Comment (styfle) [inline .github/CODEOWNERS]: No more code owners?

File: gatsby/package.json [hunk 2]
```diff
@@ -1,30 +1,48 @@
 {
   "name": "gatsby-starter-default",
+<<<<<<< HEAD
```
Comment (Timer) [inline gatsby/package.json]: 🚨

File: nextjs/package.json [hunk 3]
```diff
@@ -1,13 +1,17 @@
 {
+<<<<<<< HEAD
```
Comment (Timer) [inline nextjs/package.json]: 🚨

File: gatsby/package.json (hunk 2 above)
Comment (msweeneydev) [reply gatsby/package.json]: I got them, thanks 👍

File: nextjs/README.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ This directory is a brief example of a [Next.js](https://nextjs.org) app that ca
 
 ## Initializing this Example
 
-To get started with Next.js on Now, you can use the [Now CLI](https://zeit.co/download) to initialize the project:
+To get started with Next.js on Now, you can use the [Create-Next-App CLI](https://open.segment.com/create-next-app/) to initialize the project:
```
Comment (timneutkens) [inline nextjs/README.md]: This needs to be updated

**Comment by dolphin278** [comment]
May I ask a question – why do you remove all now.json files and `monorepo` example completely? Will there be any new configuration mechanism instead of now.json or something, that renders now.json irrelevant?

**Comment by rauchg** [comment]
Hey @dolphin278 – we don't think people should *need* any configuration to get started, so we came up with a convention.

Furthermore, all the examples that end in `+functions` fit our definition of monorepo. All the files contained in `api/` become functions (in whatever language you specify), and the top level directory is the frontend of your app.

**Comment by dolphin278** [comment]
@rauchg thank you for explanation. Looks like major simplification for app development. 

Does it mean that now.json-based routing in it's current form will be used only for specific use cases, like setting up cache headers, static redirects, etc (besides project-wide settings like aliases, regions, etc).?

**Pull request merged**